frontend/components/NewItem/NewItemForm/index.js

```diff
@@ -59,32 +106,50 @@ const styles = StyleSheet.create({
 	input: {
 		width: '90%',
 		marginLeft: '5%',
+		backgroundColor: 'white',
 	},
 	categoriesSelector: {
 		// borderWidth: 2,
 		// borderColor: 'black',
 		height: 150,
 		marginLeft: '5%',
-    marginRight: '5%',
-    marginTop: '3%'
+		marginRight: '5%',
+		marginTop: '3%',
+		justifyContent: 'space-evenly',
 	},
 	categoriesText: {
 		fontWeight: 'bold',
 		fontSize: 18,
-		justifyContent: 'flex-start',
 	},
 	itemEditors: {
 		borderRadius: 12,
 		width: '90%',
 		marginLeft: '5%',
-		padding: 20,
+		paddingLeft: '3%',
+		paddingRight: '3%',
+		marginTop: '3%',
+		flexDirection: 'row',
+		justifyContent: 'space-between',
+		alignItems: 'center',
+		height: 70,
+	},
+	itemDropDown: {
```

Comment:
For each of these, let's nest styles further by category. For example, if I want styles for `styles.itemEditors` we should access it with `styles.item.editors`, or if I want `styles.categoriesSelector` access it via `styles.categories.selector`. This will help with code readability.